src/configure/helper/remoteServiceUrlHelper.ts

```diff
@@ -18,6 +18,14 @@ export class RemoteServiceUrlHelper {
     public static provisioningServiceRedirectUrl: string = "https://go.microsoft.com/fwlink/?linkid=2142042";
 
     public static async getTemplateServiceDefinition(): Promise<IServiceUrlDefinition> {
+        const deployment = process.env["DEPLOY_TO_AZURE_ENVIRONMENT"];
```

Comment:
A more precise name would be `DEPLOY_ENVIRONMENT` as Azure is just another environment

Reply:
I have changed it to DEPLOY_TO_AZURE_EXT_ENVIRONMENT  because DEPLOY_ENVIRONMENT is little common name so there may be change if any user is using it and it has this env set